### ISSUE TITLE
BRATS Dataset training testing split

### ISSUE THREAD

**Issue by rtx98** [WuJunde/MedSegDiff#22]
Hi there, nice work.
Can you provide me your training and testing split for the BRATS21 dataset? I am trying to reproduce your work so I would like to know how to create the actual samples I need to train and infer upon. In the paper you wrote `Train/validation/test sets are split following the default settings of the dataset` , but their validation split sets don't have labels. Can you tell me how to find them?

Also did you do any preprocessing except slicing the images from 3D to 2D?

**Comment by saisusmitha** [comment]
@rtx98 did you find the answer for your question?

**Comment by cqlouis** [comment]
> Hi there, nice work. Can you provide me your training and testing split for the BRATS21 dataset? I am trying to reproduce your work so I would like to know how to create the actual samples I need to train and infer upon. In the paper you wrote `Train/validation/test sets are split following the default settings of the dataset` , but their validation and test split sets don't have labels. Can you tell me how to find them?
> 
> Also did you do any preprocessing except slicing the images from 3D to 2D?

For my understanding, the training and testing split was done on the original Training data, so you do have labels. 
But I'm not 100% sure about my point.
